tools/AzureRM.BootStrapper/ProfileMap.json

```diff
@@ -1,62 +1,64 @@
 {
 	"latest": {
-		"AzureRM": ["4.4.1"],
-		"AzureRM.Profile": ["3.4.1"], 
-		"Azure.Storage": ["3.4.1"],
-		"AzureRM.AnalysisServices": ["0.4.7"], 
-		"Azure.AnalysisServices": ["0.4.7"], 
-		"AzureRM.ApiManagement": ["4.4.1"], 
-		"AzureRM.Automation": ["3.4.1"], 
-		"AzureRM.Backup": ["3.4.1"], 
-		"AzureRM.Batch": ["3.4.1"], 
-		"AzureRM.Billing": ["0.13.7"], 
-		"AzureRM.Cdn": ["3.4.1"], 
-		"AzureRM.CognitiveServices": ["0.8.7"], 
-		"AzureRM.Compute": ["3.4.1"],
-		"AzureRM.Consumption": ["0.2.7"],
-		"AzureRM.ContainerInstance": ["0.0.2"],
-		"AzureRM.ContainerRegistry": ["0.2.7"],
-		"AzureRM.DataFactories": ["3.4.1"], 
-		"AzureRM.DataFactoryV2": ["0.2.1"], 
-		"AzureRM.DataLakeAnalytics": ["3.4.1"], 
-		"AzureRM.DataLakeStore": ["4.4.1"], 
-		"AzureRM.DevTestLabs": ["3.4.1"], 
-		"AzureRM.Dns": ["3.4.1"], 
-		"AzureRM.EventGrid": ["0.1.1"],
-		"AzureRM.EventHub": ["0.4.7"], 
-		"AzureRM.HDInsight": ["3.4.1"], 
-		"AzureRM.Insights": ["3.4.1"], 
-		"AzureRM.IoTHub": ["2.4.1"], 
-		"AzureRM.KeyVault": ["3.4.1"], 
-		"AzureRM.LogicApp": ["3.4.1"], 
-		"AzureRM.MachineLearning": ["0.15.7"], 
-		"AzureRM.MachineLearningCompute": ["0.1.0"], 
-		"AzureRM.Media": ["0.7.7"], 
-		"AzureRM.Network": ["4.4.1"], 
-		"AzureRM.NotificationHubs": ["3.4.1"], 
-		"AzureRM.OperationalInsights": ["3.4.1"], 
-		"AzureRM.PowerBIEmbedded": ["3.4,1"], 
-		"AzureRM.RecoveryServices": ["3.4.1"], 
-		"AzureRM.RecoveryServices.Backup": ["3.4.1"], 
-		"AzureRM.RedisCache": ["3.4.1"], 
-		"AzureRM.Relay": ["0.2.7"], 
-		"AzureRM.Resources": ["4.4.1"], 
-		"AzureRM.Scheduler": ["0.15.7"], 
-		"AzureRM.ServerManagement": ["3.4.1"], 
-		"AzureRM.ServiceBus": ["0.4.7"], 
-		"AzureRM.ServiceFabric": ["0.2.7"], 
-		"AzureRM.SiteRecovery": ["4.4.1"], 
-		"AzureRM.Sql": ["3.4.1"], 
-		"AzureRM.Storage": ["3.4.1"], 
-		"AzureRM.StreamAnalytics": ["3.4.1"], 
-		"AzureRM.Tags": ["3.4.1"], 
-		"AzureRM.TrafficManager": ["3.4.1"], 
-		"AzureRM.UsageAggregates": ["3.4.1"], 
-		"AzureRM.Websites": ["3.4.1"] 
+		"AzureRM": ["6.7.0"],
+		"AzureRM.Profile": ["5.4.0"], 
```

Comment:
Is this the right version for latest?

Reply:
Should be 5.4.1!

Reply:
It was when I created the PR. Will update.